packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -667,7 +672,7 @@ export class ObjectSetListenerWebsocket {
 /** @internal */
 export function constructWebsocketUrl(
   baseUrl: string,
-  ontologyRid: string | Promise<string>,
+  ontologyRid: string,
```

Comment:
Caught an actual bug. The error was on line 679 because `...${ontologyRid}...` is a problem if its a promise!